src/plus/webviews/graph/graphWebview.ts

```diff
@@ -1652,6 +1653,7 @@ export class GraphWebview extends WebviewBase<State> {
 			highlightRowsOnRefHover: configuration.get('graph.highlightRowsOnRefHover'),
 			minimap: configuration.get('graph.experimental.minimap.enabled'),
 			scrollRowPadding: configuration.get('graph.scrollRowPadding'),
+			showScrollMarkers: configuration.get('graph.scrollMarkers.enabled'),
```

Comment:
Think we should only passthrough the `enabledScrollMarkers` array -- e.g. convert our settings into the Graph version

Reply:
Done in commit: [Use markers in settings](https://github.com/gitkraken/vscode-gitlens/pull/2489/commits/780016d774cce7e278eb5652ce8499d4bd5dbc51)